src/components/messenger/list/conversation-list-panel/index.tsx

```diff
@@ -26,7 +26,7 @@ export interface Properties {
   search: (input: string) => any;
   startConversation: () => void;
   onCreateConversation: (userId: string) => void;
-  onConversationClick: (conversationId: string) => void;
+  openConversation: (payload: { conversationId: string }) => void;
```

Comment:
I think this still makes more sense as an event rather than a direct function call. At this level of component we should couple ourselves to the format of the redux actions. I.E., `onConversationClick` is still ok. (if anything, `onConversationSelected` since that's really what the user is doing)